README.md

```diff
@@ -46,3 +47,8 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+- Worst Case Memory Complexity : $O(n!)$
+* our cities array can have n! permutations, therefore our cache will need to store n! items. 
```

Comment:
Are you storing permutations in your cache or something else?

Reply:
Have another look at this. If you have two different permutations of the same set of cities, should and would that result in the same or different cache entries?